lib/sycamore/sycamore/evaluation/subtasks.py

```diff
@@ -0,0 +1,139 @@
+import json
+from typing import Any, Optional
+from sycamore.data.document import Document, OpenSearchQuery
+from sycamore.data.element import Element
+from sycamore.docset import DocSet
+from sycamore.evaluation.data import EvaluationDataPoint
+from sycamore.llms.openai import OpenAI, OpenAIModels
+from sycamore.llms.prompts.default_prompts import TaskIdentifierZeroShotGuidancePrompt
+from sycamore.transforms.embed import Embedder, SentenceTransformerEmbedder
+from sycamore.transforms.query import QueryExecutor
+
+openai_llm = OpenAI(OpenAIModels.GPT_3_5_TURBO.value)
+prompt = TaskIdentifierZeroShotGuidancePrompt()
```

Comment:
These should be overrideable - suggested change is put these in the `__init__` args as default values and save them as instance variables.